addons/web/models/models.py

```diff
@@ -347,7 +347,7 @@ def group_id_name(value):
                     for group in groups
                 }
             # retrieve all possible values, and return them with their label and counter
-            selection = self.fields_get([field_name])[field_name]
+            selection = self.fields_get([field_name])[field_name]['selection']
```

Comment:
missing test

Reply:
no test here, the test module will appear in master